examples/nextjs-14-sensitive-info/app/api/arcjet/route.ts

```diff
@@ -0,0 +1,36 @@
+import arcjet, { sensitiveInfo } from "@arcjet/next";
+import { NextResponse } from "next/server";
+
+const aj = arcjet({
+  // Get your site key from https://app.arcjet.com
+  // and set it as an environment variable rather than hard coding.
+  // See: https://nextjs.org/docs/app/building-your-application/configuring/environment-variables
+  key: process.env.ARCJET_KEY!,
+  rules: [
+    // Allows email addresses to be submitted and blocks all other types of sensitive information
+    sensitiveInfo({
+      allow: ["email"], // Will block all sensitive information types other than email.
+      // deny: ["credit-card-number", (tokens) => { return new Array(tokens.length).fill("custom") }], // Will block all sensitive information types other than email.
```

Comment:
This commented example is outdated with the new custom detect API.

Reply:
I've replaced this with a much better example